crates/polars-stream/src/graph.rs

```diff
@@ -82,14 +82,16 @@ impl Graph {
             send_state.extend(node.outputs.iter().map(|o| self.pipes[*o].recv_state));
 
             // Compute the new state of this node given its environment.
-            // println!("updating {}, before: {recv_state:?} {send_state:?}", node.compute.name());
+            // eprintln!("updating {}, before: {recv_state:?} {send_state:?}", node.compute.name());
```

Comment:
Let's just uncomment this and put behind a verbose branch.

Reply:
It's too noisy even for verbose, it's only for active debugging.